pkg/dashboard/ui/package.json

```diff
@@ -48,7 +48,7 @@
     "i18next-chained-backend": "^3.0.2",
     "i18next-localstorage-backend": "^3.1.3",
     "i18next-xhr-backend": "^3.2.2",
-    "iguazio.dashboard-controls": "^1.0.11-1.13.x",
+    "iguazio.dashboard-controls": "1.0.11-1.13.x",
```

Comment:
from what i remember, it should be less than "iguazio.dashboard-controls": "1.2.0" (because that's the version in development branch). Doesn't it?